packages/react/src/Tabs.ts

```diff
@@ -0,0 +1 @@
+export * from '@fluentui/react-tabs/lib/Tabs';
```

Comment:
let's not export `Tabs` yet until the all the renaming are done and we are ready to ship with it without futher breaking changes. Also, how are you planning to rename the props? `Pivot` needs to stay as is.